src/device/utilities.js

```diff
@@ -290,16 +290,13 @@ realityEditor.device.utilities.diffArrays = function(oldArray, newArray) {
     }
 };
 
-// speeding things up by only verifying once
-realityEditor.device.utilities.isItADesktop = window.navigator.userAgent.indexOf('Mobile') <= -1 || window.navigator.userAgent.indexOf('Macintosh') > -1;
-
 /**
  * Utility function to determine if the editor is loaded on a desktop browser or within the full mobile app.
  * @todo: make more robust so that loading on a mobile safari browser is distinguisable from within the Reality Editor app
  * @return {boolean}
  */
 realityEditor.device.utilities.isDesktop = function() {
-    return realityEditor.device.utilities.isItADesktop;
+    return false;
```

Comment:
This can just be removed with the new env system, right?

Reply:
good catch - will remove!